src/pushsource/_impl/model/ami.py

```diff
@@ -99,6 +99,21 @@ class AmiPushItem(VMIPushItem):
     )
     """Billing codes associated with this image."""
 
+    description = attr.ib(type=str, default=None, validator=optional_str)
+    """A description for the image"""
+
+    release_notes = attr.ib(type=str, default=None, validator=optional_str)
+    """"Release notes for AMI """
+
+    usage_instructions = attr.ib(type=str, default=None, validator=optional_str)
+    """Instructions for use of AMI"""
+
+    recommended_instance_type = attr.ib(type=str, default=None, validator=optional_str)
+    """Recommended instance type for AMI"""
```

Comment:
And this is also not entirely clear.

If this is the same thing as https://aws.amazon.com/ec2/instance-types/ , probably listing an example or two would be enough to make that clear, e.g. "m6in.8xlarge".

Reply:
Yes, It is same as ec2 instance-type, I will update it with some references.